data/tags.yml

```diff
@@ -343,3 +343,6 @@
 - name: YAML
   value: yaml
   type: other
+- name: Keploy
+  value: keploy
+  type: other
```

Comment:
This needs to be removed as we don't introduce new tags for tools. There should be more than a single tool per tag. Guess that was a misunderstanding of the file's purpose. 😉